README.md

```diff
@@ -47,19 +47,19 @@ Alfa will run in any [ECMAScript 2015](https://www.ecma-international.org/ecma-2
 
 ## Building
 
-In order to build Alfa, a recent version (>= 9) of [Node.js](https://nodejs.org/) is required in addition to the [Yarn](https://yarnpkg.com/) package manager. For builds, [npm](https://www.npmjs.com/) is **not** supported as Alfa makes use of [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). Once Node.js and Yarn have been installed, go ahead and install the Alfa development dependencies:
+In order to build Alfa, a recent version (>= 9) of [Node.js](https://nodejs.org/) is required in addition to the [Yarn](https://yarnpkg.com/) package manager. For builds, [npm](https://www.npmjs.com/) is **not** supported as Alfa makes use of [Yarn workspaces](https://yarnpkg.com/lang/en/docs/workspaces/). Once Node.js and Yarn have is installed, go ahead and install the Alfa development dependencies:
 
 ```console
 $ yarn install
 ```
 
-The above will also perform an initial build of Alfa. When done, you can start a file watcher which will watch source files for changes and kick off the associated build steps when they change:
+The above also performs an initial build of Alfa. When done, you can start a file watcher which watches source files for changes and kick off the associated build steps when they change:
 
 ```console
 $ yarn start
 ```
 
-As new code is pulled from origin, changes to dependencies and code may require you to run the installation again or, if only code has changed, a full build:
+If new commits pulled from this repository modifies any dependencies, you might have to rerun the installation. If only code within Alfa has changed, only a full build is required:
```

Comment:
Preferred the old version

Reply:
The old version is unclear. What does "changes to dependencies" mean? What does "If only code has changed" mean?

If a dependency has been updated/changed, nothing happens unless we modify the `lockfile` within Alfa. Also, our dependencies are written using code, so "only code has changed" makes little sense.

Lastly, origin is not the correct terminology, as it is an alias, not a property.

Reply:
I'd change `origin` to `the repository` and leave it at that; once people have made it this far in the setup process, it's fair to assume that they understand basic software and git terminology.